src/alpaka/AlpakaCore/alpakaMemory.h

```diff
@@ -75,12 +78,19 @@ namespace cms::alpakatools {
   template <typename TBuf>
   void pin_buffer(TBuf& buffer) {
 #ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
-    if (not devices<alpaka::PltfUniformCudaHipRt>.empty()) {
-      // it is possible to initialise the CUDA runtime and call cudaHostRegister
-      // only if the system has at least one supported GPU
-      alpaka::prepareForAsyncCopy(buffer);
-    }
+      if (not cms::alpakatools::devices<alpaka::PltfCudaRt>.empty()) {
```

Comment:
Why the extra indentation? Does it come from `clang-format`?

Reply:
Mhm, I guess... let me check

Reply:
Actually, either I didn't run it, or I ran some different version... I've just pushed the changes from `clang-format-12`